cmd/rigs/cmd/publish_clear-data.go

```diff
@@ -16,7 +16,9 @@ func init() {
 
 	clearDataCmd.Flags().Bool("parts", false, "clear data for the parts table")
 	clearDataCmd.Flags().Bool("layers", false, "clear data for the layers table")
+	clearDataCmd.Flags().Bool("rigs", false, "clear data for the rigs table")
```

Comment:
Throughout the PR there are many mechanical changes like this just to accommodate the new rigs and deals tables. Don't worry too much about these types of changes.